CHANGES.txt

```diff
@@ -5,10 +5,20 @@ Changes for crate
 Unreleased
 ==========
 
-- Added deprecation warning about dropping support for SQLAlchemy 1.3 soon, it is
-  effectively EOL.
 
-- Added support for SQLAlchemy 2.0.
+2023/02/16 0.30.0
+=================
+
+- Added deprecation warning about dropping support for SQLAlchemy 1.3 soon, it
+  is effectively EOL.
+
+- Added support for SQLAlchemy 2.0. See also `What's New in SQLAlchemy 2.0`_
+  and `SQLAlchemy 2.0 migration guide`_.
+
+- Updated to geojson 3.0.0.
```

Comment:
correct? just double checking, as this was not in the unreleased section before.

Reply:
Yes. I've enriched it a bit.